### PR TITLE
openstack UPI: Small documentation fixes

### PR DESCRIPTION
**Pull request by pierreprinetti**
* Normalise title hierarchy
* Incorporate Glance instructions into the flow
* TLS public certificates are not sensitive data

/label platform/openstack
/cc @mandre @MaysaMacedo

### CODE REVIEW

File: docs/user/openstack/install_upi.md [hunk 1]
```diff
@@ -552,48 +550,19 @@ To obtain the token execute:
...
-    "version": "2.4.0"
-  },
-  "networkd": {},
-  "passwd": {},
-  "storage": {},
-  "systemd": {}
-}
-```
+Combine the public URL with the `file` value from the previous step to get the link to your bootstrap ignition.
+Example of the link: `https://public.glance.example.com:9292/v2/images/b7e2b84e-15cf-440a-a113-3197518da024/file`.
```
Comment (MaysaMacedo) [inline docs/user/openstack/install_upi.md:565]: perhaps it's good to mention this link would need to be added to the source field on the Ignition Shim

**Comment by pierreprinetti** [comment]
Thank you @MaysaMacedo. I have also tried to streamline the instructions for the Swift filename, which were oddly intertwined with Glance's after this change.

**Pull request merged**